src/vm/database/sqlite.rs

```diff
@@ -22,6 +29,15 @@ pub struct ContractDatabase <'a> {
     savepoint: Savepoint<'a>
 }
 
+pub struct SqliteToken {
+    token_identifier: i64
```

Comment:
Is the order in which these identifiers are allocated stable across forks?  i.e. if a token has identifier X, then does that mean that (1) there are X - 1 prior tokens defined in the fork in which the Xth token type was defined, and (2) all peers in the same fork will have the same token definition for their Xth token type?  Reason I'm asking is because I was hoping to use an integer identifier for tokens and assets in the transaction wire format for encoding post-conditions in a space-efficient manner.  Same question for `SqliteAsset`.

Reply:
No -- it won't be. I think for identifying tokens, we'll need to use the contract identifier + a contract-local token identifier. The contract local identifier could be an integer, but the contract identifier itself would need to be the hash160 of the sender principal and the namespaced contract name.